pageserver/src/metrics.rs

```diff
@@ -1272,6 +1284,85 @@ impl Drop for SmgrOpTimer {
         if let Some(per_timeline_getpage_histo) = &self.per_timeline_latency_histo {
             per_timeline_getpage_histo.observe(elapsed);
         }
+
+        now
```

Comment:
nit: this includes the metric update time, but probably irrelevant in practice

Reply:
The idea was to avoid a gettimeofday call